distribution/lib/Standard/Table/0.0.0-dev/src/Value_Type.enso

```diff
@@ -452,6 +452,11 @@ type Value_Type
         if (typ == Value_Type.Null) || predicate typ then action else
             Value_Type_Helpers.raise_unexpected_type type_kind value
 
+    ## PRIVATE
+       Returns a vector of all numeric types.
+    numeric_types -> Vector Value_Type =
+        (Meta.meta Value_Type).constructors.filter t->(t.value.is_numeric) . map t->t.value
```

Comment:
This is a nice trick but it relies on the fact that all `Value_Type` constructors have only optional fields (all fields have a default value). If that were not the case, then `t.value` could not be a constructed value but only a Function and `is_numeric` call on it would fail with a `No_Such_Method`.

Probably unlikely to happen in practice, so I think this is OK, but noting that it may be brittle if we ever change `Value_Type`.

Reply:
Removed this whole method